packages/react-components/react-motion-components-preview/library/src/components/Scale/Scale.types.ts

```diff
@@ -0,0 +1,20 @@
+// eslint-disable-next-line @typescript-eslint/naming-convention
+export type ScaleVariantParams_unstable = {
```

Comment:
Good choice to mark these as unstable in the naming convention, since there is more design work that needs to be done.